gateway-service/src/main/java/org/zowe/apiml/gateway/config/WebSecurity.java

```diff
@@ -322,7 +324,9 @@ public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http, Au
                 SERVICES_SHORT_URL,
                 SERVICES_SHORT_URL + "/**",
                 SERVICES_FULL_URL,
-                SERVICES_FULL_URL + "/**"
+                SERVICES_FULL_URL + "/**",
+                CONFORMANCE,
+                VALIDATE
```

Comment:
This is the part that's not yet working right? maybe add a TODO ?

Reply:
Yes, good point